src/test/java/com/wikia/webdriver/common/dataprovider/mobile/MobileAdsDataProvider.java

```diff
@@ -63,9 +93,271 @@ private MobileAdsDataProvider() {
                 "\"s2\":\"article\"",
                 "\"dmn\":\"wikiacom\"",
                 "\"hostpre\":\"",
+                "\"ab\":[",
+                "\"ksgmnt\":[",
+                "\"u\":\"",
+                "\"dmn\":\"",
+                "\"skin\":\"mercury\"",
                 "\"wpage\":\"synthetictests/dfpparams\"",
                 "\"ref\":\"direct\"",
                 "\"lang\":\"en\"",
+                "\"skin\":\"mercury\"",
+                "\"esrb\":\"teen\""
+
+            ),
+            Arrays.asList(
+                "\"pos\":\"MOBILE_TOP_LEADERBOARD\"",
+                "\"src\":\"mobile\""
+            )
+        },
+        {
+            "yugioh",
+            "Dark_Magician",
+            "wka.ent/_yugioh//article",
+            "MOBILE_TOP_LEADERBOARD",
+            Arrays.asList(
+                "\"s0\":\"ent\"",
+                "\"s0v\":\"comics\"",
+                "\"s0c\":[\"gaming\",\"anime\",\"videogames\"]",
+                "\"s1\":\"_yugioh\"",
+                "\"s2\":\"article\"",
+                "\"dmn\":\"wikiacom\"",
+                "\"hostpre\":\"yugioh\"",
+                "\"ab\":[",
+                "\"dmn\":\"",
+                "\"skin\":\"mercury\"",
+                "\"ar\":\"4:3\"",
+                "\"artid\":\"278\"",
+                "\"wpage\":\"dark_magician\"",
+                "\"ref\":\"direct\"",
+                "\"lang\":\"en\"",
+                "\"pv\":\"1\"",
+                "\"top\":\"1k\"",
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "\"top\":\"1k\"" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)